interface/modules/custom_modules/oe-module-faxsms/templates/phone_widget.html.twig

```diff
@@ -0,0 +1,26 @@
+
+<div id="rc-container" class="d-flex flex-row align-items-center justify-content-between">
+    <div id="rc-header">
+        <script>
+            if ('setSinkId' in HTMLMediaElement.prototype) {
+                HTMLMediaElement.prototype.setSinkId = function(deviceId) {
+                    return new Promise((resolve, reject) => {
+                        if (deviceId && typeof deviceId === 'string') {
+                            // Simulate setting the sink ID
+                            console.log(`Setting sink ID to: ${deviceId}`);
+                            resolve();
+                        } else {
+                            reject(new Error('Invalid device ID'));
+                        }
+                    });
+                };
+            }
+            (function () {
+                const rcs = document.createElement("script");
+                rcs.src = "https://apps.ringcentral.com/integration/ringcentral-embeddable/3.x/adapter.js?enableNoiseReductionSetting=1&enableSideWidget=1&clientId={{ clientId }}&clientSecret={{ clientSecret }}&jwt={{ jwt }}";
```

Comment:
above need js escaping and url encoding. something like:
```
rcs.src = "https://apps.ringcentral.com/integration/ringcentral-embeddable/3.x/adapter.js?enableNoiseReductionSetting=1&enableSideWidget=1&clientId=" + {{ clientId|js_url }} + "&clientSecret=" + {{ clientSecret|js_url }} + "&jwt=" + {{ jwt|js_url }};
```

Reply:
It's already urlencoded when passed in and embedded in a string so needs no more escaping.